book/src/feature/logging.md

```diff
@@ -162,4 +162,4 @@ class _MyHomePageState extends State<MyHomePage> {
     );
   }
 }
-```
\ No newline at end of file
+```
```

Comment:
I don't know what changed here :D

Reply:
I also dnk... maybe just revert it

Reply:
the change is that there was no newline at the end of the file. Might be worth it to keep the change.